src/bin/pg_autoctl/keeper_pg_init.c

```diff
@@ -155,6 +155,14 @@ keeper_pg_init_and_register(Keeper *keeper)
 		return keeper_init_fsm(keeper);
 	}
 
+	char scrubbedConnectionString[MAXCONNINFO] = { 0 };
+	if (!parse_and_scrub_connection_string(config->monitor_pguri,
+										   scrubbedConnectionString))
+	{
+		log_error("Monitor connection string is unparseable");
```

Comment:
Better written in the same style as the other errors messages, such as: "Failed to parse the monitor connection string".